cardano-explorer-node/src/Explorer/Node.hs

```diff
@@ -152,7 +152,7 @@ initializeAllFeatures enp = do
 commonCli :: ExplorerNodeParams -> CommonCLI
 commonCli enp =
   CommonCLI
-    { Config.cliSocketPath = Last $ Just (unSocketPath $ enpSocketPath enp)
+    { Config.cliSocketDir = Last $ Just (unSocketPath $ enpSocketPath enp)
```

Comment:
in `cardano-node`, its `sockerDir`, because the name of the socket is dynamically generated, based on other flags
but in explorer, we dont care about what those flags are, and just want a path to the socket itself

and i suspect that the node client doesnt need a socket path at all since we are passing the correct socket to `connectTo`
what happens if this field is just set to `Nothing` ?

Reply:
You're right. I will fix that.